internal/proto/common.proto

```diff
@@ -65,6 +65,7 @@ enum SegmentState {
     Flushing = 5;
     Dropped = 6;
     Importing = 7;
+    Compacted = 8;
```

Comment:
do we really need compacted state?
Dropped state should be good enough,
File can not be removed under dropped state